atom/browser/native_window_mac.mm

```diff
@@ -1359,23 +1359,6 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 }
 
 void NativeWindowMac::AddContentViewLayers() {
-  // Make sure the bottom corner is rounded for non-modal windows:
-  // http://crbug.com/396264. But do not enable it on OS X 10.9 for transparent
-  // window, otherwise a semi-transparent frame would show.
-  if (!(transparent() && base::mac::IsOS10_9()) && !is_modal()) {
```

Comment:
🚨this changes behavior on macOS 10.10, which I don't think you intend.

```
!(transparent() && base::mac::IsOS10_9()) && !is_modal()
==>
!(transparent() && false) && !is_modal()
==>
!(false) && !is_modal()
==>
true && !is_modal()
==>
!is_modal()
```

Reply:
oops missed a paren, thought it was three equal condition!